[Bug]: ERC20 Token symbol displays 'undefined' in send confirmation screen, amount also displays 0

### Describe the bug


https://user-images.githubusercontent.com/53189696/231795314-f75d9090-f9c6-41c4-bc3c-6b3e3320af12.mov



### Steps to reproduce

1. Log in to the extension
2. navigate to the test dapp
3. create TST token
4. add TST token to the wallet
5. Send Token


### Error messages or log output

```shell
token-util.js:260 Error: Unable to determine contract standard
  at p.<anonymous> (AssetsContractController.js:170:1)
  at Generator.throw (<anonymous>)
  at a (AssetsContractController.js:6:1)
a
```


### Version

10.29

### Build type

None

### Browser

Chrome

### Operating system

MacOS

### Hardware wallet

_No response_

### Additional context

_No response_

--- a/ui/pages/confirm-token-transaction-base/confirm-token-transaction-base.js
+++ b/ui/pages/confirm-token-transaction-base/confirm-token-transaction-base.js
@@ -70,6 +70,7 @@ export default function ConfirmTokenTransactionBase({
       (collection) =>
         collection.address.toLowerCase() === tokenAddress.toLowerCase(),
     );
+    console.log({ tokenSymbol, nftCollection })
     const titleTokenDescription =
       tokenSymbol || nftCollection?.name || t('unknownCollection');
 
@@ -126,6 +127,7 @@ export default function ConfirmTokenTransactionBase({
       assetName || `${getTitleTokenDescription('text')} #${tokenId}`;
   } else if (assetStandard === TokenStandard.ERC20) {
     title = `${tokenAmount} ${tokenSymbol}`;
+    subtotalDisplay = `${tokenAmount} ${tokenSymbol}`;
   }
 
   const hexWeiValue = useMemo(() => {

--- a/ui/pages/confirm-token-transaction-base/confirm-token-transaction-base.js
+++ b/ui/pages/confirm-token-transaction-base/confirm-token-transaction-base.js
@@ -70,7 +70,6 @@ export default function ConfirmTokenTransactionBase({
       (collection) =>
         collection.address.toLowerCase() === tokenAddress.toLowerCase(),
     );
-    console.log({ tokenSymbol, nftCollection })
     const titleTokenDescription =
       tokenSymbol || nftCollection?.name || t('unknownCollection');
 

